formula/src/main/java/com/instacart/formula/FormulaRuntime.kt

```diff
@@ -153,4 +153,8 @@ class FormulaRuntime<Input : Any, State, RenderModel : Any>(
             }
         }
     }
+
+    private fun <T> checkAnyNotNull(value: T?): T {
+        return checkNotNull(value as Any?) as T
+    }
```

Comment:
This check is needed, since we have to change `Input : Any` and `RenderModel : Any`, to make `FormulaRuntime` compatible with the types defined within `Formula`. Otherwise, we would have to change over to `Input : Any` and `RenderModel : Any` in all classes that extend `Formula`.